Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py

```diff
@@ -609,6 +609,14 @@ def branches_for(self, hash=None, remote=True):
             return sorted(result.get(remote, []))
         return result
 
+    def is_pull_request_branch(self, branch, source_remote):
+        if branch is None or branch in self.DEFAULT_BRANCHES or self.PROD_BRANCHES.match(branch):
+            return False
+        elif branch in self.branches_for(remote=source_remote):
+            if not self.dev_branches.match(branch):
+                return False
```

Comment:
No reason to have nested conditionals here:

```suggestion
        elif branch in self.branches_for(remote=source_remote) and not self.dev_branches.match(branch):
            return False
```